tensorflow/compiler/xla/service/gpu/ir_emitter.cc

```diff
@@ -50,6 +50,23 @@ limitations under the License.
 #include "tensorflow/compiler/xla/window_util.h"
 #include "tensorflow/core/lib/core/errors.h"
 
+// Convenient function to cast the provided llvm::Value* using IRBuilder
+// to default address space. This is useful in particular in generating
+// IR for AMDGPU target, as its kernel variables is in address space 5
```

Comment:
"for generating", "are in address space", "instead of the default"

Reply:
Ooops thanks for pointing out.